backend/graminsta/core/views.py

```diff
@@ -1,3 +1,40 @@
 # -*- coding: UTF-8 -*-
 
-# Create your views here.
+"""
+Register and Login.
+"""
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from .models import UserInfo
+from .serializers import UserInfoSerializer
+
+
+class UserInfoRecordView(APIView):
```

Comment:
Based on my previous commment, in thest `get` and `post` handlers, you will call APIs exposed by `services.py`, instead of interacting with the `models.py` directly.

To be more specific:

views.py: Handles user requests and invokes APIs exposed by services.py
services.py: Each API in this service tries to perform a single operation (serialization/deserailization is done by serializers.py), like `user registration`, `create new post`, etc.
serializers.py: Handles serialization/deserialization logic on models.
models.py: Interacts with the database.